docs/translations.md

```diff
@@ -1,43 +1,777 @@
+Germany's prices in German and Euros. A minority are more complex, choosing a price
+based on language. Belgium users share the price in Euros of a neighbor, based on
+language. Switzerland has separate prices for German, French, and Italian speakers.
+All the Swiss prices are in Swiss Francs. See [privaterelay/plans.py][] for details.
+
+In development and stage, there are test prices connected to the [Stripe test mode][].
+One feature of test mode is fake credit cards. These cards allow testing without paying
+real money, and testing failure modes. A test visitor from the United States will get
+these prices. Other regions get the production prices.
+
+[/api/v1/runtime_data]: https://relay.firefox.com/api/v1/runtime_data
+[Stripe product]: https://stripe.com/docs/api/products
+[Stripe test mode]: https://stripe.com/docs/test-mode
+
+## Terms
+
+Relay developers will see these terms as they work in this topic:
+
+- **[Internationalization][]**: Designing a service to adapt to different locales
+  without per-locale code changes. This is often abbreviated **i18n**, to stand for the
+  starting letter "i", the next 18 letters, and the ending letter "n". This abbreviation
+  also avoids the spelling differences between American and British English.
+- **[Localization][]**: Adapting software to a specific locale. This includes translating
+  text and using locale-specific formats. This is often abbreviated **L10n**, in a
+  similar way to i18n. A capital "L" avoids confusing a lowercase "l" with an
+  uppercase "I".
+- **[Locale][]**: Shared cultural conventions that appear in a user interface. This
+  can include language, letters, and currencies. It can also include formats for prices,
+  numbers, dates, and times. For example, American English can be a locale. This locale
+  uses the English alphabet in left-to-right text. Prices are in US Dollars (USD, "$5.99").
+  Numbers use commas as thousands separators (525,960 minutes in a year). The first
+  number in a short date is the month ("9/12/2023" for September 12, 2023).
+  English-speaking users in other regions share some of these conventions. Other
+  conventions, like currency and date formats, will vary.
+- **[Region][]**: An [administrative division][] that can be the basis for a
+  locale. Mozilla prefers to use "Region" instead of "**Country**" when talking about
+  localization. A region can include areas within a country, as well as areas that
+  span countries.
+- **[Language][]**: A spoken or written system of communication. English is a broad
+  language category. A language can be specific to a region, such as German spoken in
+  Switzerland. In localization, the written system can be important as well. For
+  example, Chinese can vary between mainland China and other regions. It can also
+  use Simplified or Traditional Chinese script when written.
+- **[Translation][]**: The process of adapting language strings or patterns to a
```

Comment:
(suggestion, non-blocking) Add `Subscription` or `Subscription Platform` section here to describe our subscription platform as a service.